README.md

```diff
@@ -48,6 +48,15 @@ wash app deploy ./helloworld/wadm.yaml
 curl localhost:8080
 ```
 
+In a scenario where you are behind a proxy, you can set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables to the proxy URL.
+And if your proxy requires authentication, you can set the `WASH_PROXY_USERNAME` and `WASH_PROXY_PASSWORD` environment variables to the username and password, respectively. Since most passwords contain special characters, you will need to specify the value for 'WASH_PROXY_PASSWORD' in single quotes.
+eg:
+
+```console
+export WASH_PROXY_USERNAME='proxyuser' 
+export WASH_PROXY_PASSWORD='proxypass'
+```
```

Comment:
```suggestion

## Proxy authentication
In a scenario where you are behind a proxy, you can set the `HTTP_PROXY` and `HTTPS_PROXY` environment variables to the proxy URL.
And if your proxy requires authentication, you can set the `WASH_PROXY_USERNAME` and `WASH_PROXY_PASSWORD` environment variables to the username and password, respectively. Since most passwords contain special characters, it's recommended to specify the value for 'WASH_PROXY_PASSWORD' in single quotes.

For example, in a unix environment:

```console
export WASH_PROXY_USERNAME='username' 
export WASH_PROXY_PASSWORD='p@ssw0rd'
```
```